### DISCUSSION TITLE
ibax game

### DISCUSSION THREAD

**Discussion by alpina1977** [IBAX-io/go-ibax#1855] [Ideas]
### Clear and concise description of the problem

Working on application improvement and development

### Suggested solution

Having game events within the ibax network application and distributing the points earned in the game to the members will help the progress and development of the system.

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by birikoray** [comment]
Good job guys

**Comment by bilalugur3** [comment]
very good

**Comment by Halo0642** [comment]
Umarım olur
